Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -596,149 +596,10 @@ public string keyboardLayout
-                    }
-                }
-
-                switch (key)
-                {
-                    case Key.Space: return spaceKey;
-                    case Key.Enter: return enterKey;
-                    case Key.Tab: return tabKey;
-                    case Key.Backquote: return backquoteKey;
-                    case Key.Quote: return quoteKey;
-                    case Key.Semicolon: return semicolonKey;
-                    case Key.Comma: return commaKey;
-                    case Key.Period: return periodKey;
-                    case Key.Slash: return slashKey;
-                    case Key.Backslash: return backslashKey;
-                    case Key.LeftBracket: return leftBracketKey;
-                    case Key.RightBracket: return rightBracketKey;
-                    case Key.Minus: return minusKey;
-                    case Key.Equals: return equalsKey;
-                    case Key.LeftShift: return leftShiftKey;
-                    case Key.RightShift: return rightShiftKey;
-                    case Key.LeftAlt: return leftAltKey;
-                    case Key.RightAlt: return rightAltKey;
-                    case Key.LeftCtrl: return leftCtrlKey;
-                    case Key.RightCtrl: return rightCtrlKey;
-                    case Key.LeftMeta: return leftMetaKey;
-                    case Key.RightMeta: return rightMetaKey;
-                    case Key.ContextMenu: return contextMenuKey;
-                    case Key.Escape: return escapeKey;
-                    case Key.LeftArrow: return leftArrowKey;
-                    case Key.RightArrow: return rightArrowKey;
-                    case Key.UpArrow: return upArrowKey;
-                    case Key.DownArrow: return downArrowKey;
-                    case Key.Backspace: return backspaceKey;
-                    case Key.PageDown: return pageDownKey;
-                    case Key.PageUp: return pageUpKey;
-                    case Key.Home: return homeKey;
-                    case Key.End: return endKey;
-                    case Key.Insert: return insertKey;
-                    case Key.Delete: return deleteKey;
-                    case Key.CapsLock: return capsLockKey;
-                    case Key.NumLock: return numLockKey;
-                    case Key.PrintScreen: return printScreenKey;
-                    case Key.ScrollLock: return scrollLockKey;
-                    case Key.Pause: return pauseKey;
-                    case Key.OEM1: return oem1Key;
-                    case Key.OEM2: return oem2Key;
-                    case Key.OEM3: return oem3Key;
-                    case Key.OEM4: return oem4Key;
-                    case Key.OEM5: return oem5Key;
-                }
-
-                throw new ArgumentException("key");
+                var index = (int)key;
+                if (index <= 0 || index > m_Keys.Length)
+                    throw new ArgumentException("key");
+                return m_Keys[(int)key];
```

Comment:
Seems like we can drop the first entry in the array altogether and just subtract one here.